setup.py

```diff
@@ -26,8 +26,10 @@
     ],
     entry_points={
         "microcosm.factories": [
+            "error_handlers = microcosm_flask.errors:configure_error_handlers",
             "health = microcosm_flask.conventions.health:configure_health",
             "flask = microcosm_flask.factories:configure_flask",
+            "app = microcosm_flask.factories:configure_flask_app",
```

Comment:
Small change here: I'm using the binding `flask` to refer to the plain Flask instance and `app` to refer to a configured instance that uses conventions. That gives the various conventions a target to configure that doesn't create a circular dependency.